packages/extension-chains/src/chains.ts

```diff
@@ -30,6 +30,12 @@ const hashes: MetadataDefBase[] = [
     genesisHash: '0xf7a99d3cb92853d00d5275c971c132c074636256583fee53b3bbe60d7b8769ba',
     icon: 'substrate',
     ss58Format: 16
+  },
+  {
```

Comment:
Can you just move this up to right under "alphabetical to avoid (further) favorites" - the alphabetical ordering makes it easier to find/maintain as the list grows.

Reply:
done